### DISCUSSION TITLE
integration yamaha receivers new generation (RX-V4A)

### DISCUSSION THREAD

**Discussion by varol56** [home-assistant/frontend#9411] [Other feature requests]
I Would like to integrate the new receiver  (RX-V4A) and have followed the integration recommendations on the HA site. 
https://www.home-assistant.io/integrations/yamaha#discovery-notes

also no error in log

dia_player:
  - platform: yamaha
    host: 192.168.0.208
    source_names:
      HDMI2: "DVH-560H"
      NET RADIO: "Internet radiol"
    zone_ignore:
      - "Zone_2"
    zone_names:
      Main_Zone: "Living C&L"

**Comment by ludeeus** [comment]
The frontend does not control integrations, please raise this on the forums <https://community.home-assistant.io/c/feature-requests/13>, thanks 👍
